src/main/java/net/neoforged/neoforge/items/ItemStackHandler.java

```diff
@@ -173,4 +181,49 @@ protected void validateSlotIndex(int slot) {
     protected void onLoad() {}
 
     protected void onContentsChanged(int slot) {}
+
+    public record ItemStackHandlerCodec(HolderLookup.Provider provider) implements Codec<ItemStackHandler> {
+
+        @Override
+        public <T> DataResult<Pair<ItemStackHandler, T>> decode(DynamicOps<T> ops, T input) {
```

Comment:
Mojang has seemingly been moving towards using snake case for all fields in codecs, so I think these fields should as well, rather than having capitals. Unless the reason for them being capitals is explicitly because `ContainerHelper` still uses the capitalized values for `Items` and `Slot`. Either way I would prefer `Size` being lowercased as there is no precedent for that, and if we decide to keep `Items` capitalized, I would prefer we reference it as `ContainerHelper.TAG_ITEMS`